frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature

```diff
@@ -0,0 +1,95 @@
+@add-flow
+Feature: Add page on Developer Console
+              As a user, I should be able select a way to create an Application, component or service from one of the options provided on Add page
+
+    
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-add-page"
+              And user has installed OpenShift Serverless Operator
+              And user has installed OpenShift Pipelines operator
+              And user is at Add page
+
+        
+        @regression
+        Scenario: Getting started resources on Developer perspective
+             Then user will see Getting started resources
+              And user will see Create Application using Samples
+              And user will see Build with guided documentation
+              And user will see Explore new developer features
+
+
+        @smoke
+        Scenario: Options to create an Application, Component and Service
+             Then user will see "Developer Catalog" card
+              And user will see "Git Repository" card
+              And user will see "Container Images" option
+              And user will see "Pipelines" option
+              And user will see "Samples" option
+              And user will see "From Local Machine" card
+              And user will see "Serverless" card
+        
+        
+        @regression
+        Scenario: Developer Catalog option to create an Application, Component and Service
+             Then user will see "All Services" option
+              And user will see "Database" option
+              And user will see "Operator Backed" option
+              And user will see "Helm Chart" option
+              And user will see "Event Source" option
+
+        
+        @regression
+        Scenario: Git Repository option to create an Application, Component and Service
+             Then user will see "From Git" option
+              And user will see "From Devfile" option
+              And user will see "From Dockerfile" option
+
+        
+        @regression
+        Scenario: From Local Machine option to create an Application, Component and Service
+             Then user will see "Import YAML" option
+              And user will see "Upload JAR file" option
+
+        
+        @regression
+        Scenario: Details Toggle on
```

Comment:
I'm not sure if it is required for a regression test, but I feel like there is no context for the description here.
Should we add which description the test is talking about?

Reply:
same goes for the "Details Toggle off" scenario

Reply:
Updated the statement which will now make sense with a scenario description.